api/external/infra_proxy/response/roles.proto

```diff
@@ -45,23 +46,6 @@ message ExpandedRunList {
     repeated RunList run_list = 2;
 }
 
-message RunList {
```

Comment:
Moved it to the `common.proto` file to be used by both `Node` and `Role` runlists

Reply:
cool